pyrightconfig.stricter.json

```diff
@@ -90,7 +90,8 @@
         "stubs/tzlocal/tzlocal/utils.pyi",
         "stubs/ttkthemes",
         "stubs/urllib3",
-        "stubs/vobject"
+        "stubs/vobject",
+        "stubs/PyAutoGUI"
```

Comment:
> We could edit the stub generating script to keep it alphabetized.

I like that idea :)

Reply:
(Actually, it looks like the stub-generating script already has logic in it to keep the list alphabetised. I'm guessing the only reason it wasn't initially alphabetised here is because of the path-separator issue being discussed in #8653.)